Cargo.toml

```diff
@@ -39,6 +39,7 @@ once_cell = "1.19.0"
 image = "0.25.1"
 reqwest = { version = "0.12.4", features = ["blocking"] }
 base64 = "0.22.1"
+bindgen_cuda = "0.1.5"
```

Comment:
It should be fine to include `bindgen_cuda` in this case because just including it doesn't do any CUDA stuff, right?

Reply:
Yes, bindgen_cuda doesn't include any CUDA stuff. And I also test the build process both on cuda and non-cuda environment. Nonetheless I did some change to make it more beautiful😂